server/computer_use/tools/extraction.py

```diff
@@ -54,6 +64,16 @@ async def __call__(self, *, data: Dict[str, Any]) -> ToolResult:
             # Log the formatted JSON
             logger.info(f'Extraction tool formatted JSON: {serialized_data}')
 
+            # validate the data adheres to the response schema
+            if self.response_schema:
```

Comment:
`response_schema` should always be defined, even when it's an empty object `{}` -> always validate.